AY2324S1-CS2103T-T12-2/tp
# Add test cases for Events

- Add test cases for editEventCommand execute
- Add EditEventCommand test cases
- Add EditCommandParserTest and error checking
- Add EditMeetingParser test cases
- Add DeleteMeetingCommandParser test

Please check the fix for assign groups before merging this :))
